sdks/python/container/boot.go

```diff
@@ -145,7 +145,21 @@ func main() {
 	// Guard from concurrent artifact retrieval and installation,
 	// when called by child processes in a worker pool.
 
+	workerPoolId := os.Getenv(workerPoolIdEnv)
+	var venvDir string
+	if workerPoolId != "" {
+		venvDir = filepath.Join(*semiPersistDir, "beam-venv", "beam-pool-" + workerPoolId)
+	} else {
+		venvDir = filepath.Join(*semiPersistDir, "beam-venv", "beam-worker-" + *id)
```

Comment:
It's not clear why there wouldn't be a workerPoolId.  Maybe add a comment.

Reply:
That depends on how the pool is started. Inside a container that will probably not happen, but the same branch existed further down in the code, so I copied it here. In this regard, a definitive answer to my previous question about whether binding the whole thing to a worker ID instead of a pool ID would be all right. That would simplify a lot of things, including this part here. The downside would be potentially redundant venvs on the same machine for different workers (I guess a tradeoff worth taking).

Reply:
Yeah, I guess I'm more making a suggestion to add a comment for less informed people like myself in the code here.  If you think someone reading this code would understand it without a comment then feel free to ignore my suggestion (I'm just saying I didn't understand).

Reply:
The question is rather whether I can bind the logic to the worker ID instead of the pool ID. Then this code would be obsolete and could be removed.

Reply:
Any opinions?

Reply:
I ran into some issues when multiple workers were spawned in the same container. I think I have to bind the venv to the worker and then clean them up individually. Otherwise some workers will get their venv deleted by other workers in the same container. I will check next week how best to solve this and update the PR.